beacon/beacon.go

```diff
@@ -813,11 +770,19 @@ func (pd *ProtocolDriver) sendFirstRoundVote(ctx context.Context, epoch types.Ep
 	return nil
 }
 
-func (pd *ProtocolDriver) sendFollowingVote(ctx context.Context, epoch types.EpochID, round types.RoundID, ownCurrentRoundVotes allVotes) error {
+func (pd *ProtocolDriver) getFirstRoundVote(minerPK *signing.PublicKey) ([][]byte, error) {
 	pd.mu.RLock()
-	bitVector := encodeVotes(ownCurrentRoundVotes, pd.firstRoundIncomingVotes[string(pd.edSigner.PublicKey().Bytes())])
-	pd.mu.RUnlock()
+	defer pd.mu.RUnlock()
+	return pd.current.getMinerFirstRoundVote(minerPK)
+}
+
+func (pd *ProtocolDriver) sendFollowingVote(ctx context.Context, epoch types.EpochID, round types.RoundID, ownCurrentRoundVotes allVotes) error {
+	firstRoundVotes, err := pd.getFirstRoundVote(pd.edSigner.PublicKey())
+	if err != nil {
+		return fmt.Errorf("failed to get own first round votes %v: %w", pd.edSigner.PublicKey().String(), err)
```

Comment:
Maybe just
```suggestion
		return fmt.Errorf("get own first round votes %v: %w", pd.edSigner.PublicKey().String(), err)
```
?

With the current text, the outer code 
```go
if err := pd.sendFollowingVote(ctx, epoch, round, votes); err != nil {
	rLogger.With().Error("failed to send following vote", log.Err(err))
}
```
would output `failed to send following vote errmsg=failed to get...` which is a bit redundant

Reply:
done